kf_lib_data_ingest/target_api_plugins/kids_first_dataservice.py

```diff
@@ -505,17 +505,11 @@ class SequencingExperiment:
     @classmethod
     def get_key_components(cls, record, get_target_id_from_record):
         return {
+            "study_id": not_none(record[CONCEPT.STUDY.TARGET_SERVICE_ID]),
             "sequencing_center_id": not_none(
                 record[CONCEPT.SEQUENCING.CENTER.TARGET_SERVICE_ID]
             ),
-            "external_id": (
-                record.get(CONCEPT.SEQUENCING.ID)
-                or not_none(record[CONCEPT.SEQUENCING.LIBRARY_NAME])
-            ),
-            "library_name": (
-                record.get(CONCEPT.SEQUENCING.LIBRARY_NAME)
-                or not_none(record[CONCEPT.SEQUENCING.ID])
-            ),
+            "external_id": not_none(record[CONCEPT.SEQUENCING.ID]),
```

Comment:
I totally agree that we shouldn't rely on other attributes even if we know that they are a 1-on-1 match.